graph.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python3
+# graph.py
+from collections import defaultdict
+
+
+class Graph():
+    def __init__(self, vertices):
+        self.graph = defaultdict(list)
+        self.V = vertices
```

Comment:
Rename both the attribute `V` and the parameter `vertices` to `num_vertices` to make it clearer.